README.md

```diff
@@ -35,19 +40,212 @@ Outline the file contents of the repository. It helps users navigate the codebas
 
 ## Prerequisites
 
-Outline the required components and tools that a user might need to have on their machine in order to run the sample. This can be anything from frameworks, SDKs, OS versions or IDE releases.
+- [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
+- [Angular-cli](https://cli.angular.io/) must be installed to run this sample.
+- [Dotnet Core SDK](https://dotnet.microsoft.com/download) must be installed to run this sample.
+- We recommend [VS Code](https://code.visualstudio.com/download) for running and debugging this cross-platform application.
 
 ## Setup

```

Comment:
>Setup [](start = 3, length = 5)

The sample name is long, so I'd suggest you add the following line first
1. Clone or download this repository

```console
git clone [https://github.com/Azure-Samples/ms-identity-javascript-angular-spa-aspnetcore-webapi.git](https://github.com/Azure-Samples/ms-identity-javascript-angular-spa-aspnetcore-webapi.git)
```
or download and extract the repository .zip file.

> Given that the name of the sample is quiet long, and so are the names of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.